scripts/helper.py

```diff
@@ -230,6 +233,62 @@ def run_fuzzer(run_args):
   pipe = subprocess.Popen(command)
   pipe.communicate()
 
+def run_with_coverage(run_args):
+  """Runs a fuzzer in the container."""
+  parser = argparse.ArgumentParser('helper.py run_with_coverage')
+  parser.add_argument('library_name', help='name of the library')
+  parser.add_argument('fuzzer_name', help='name of the fuzzer')
+  parser.add_argument('run_time', help='time in seconds to run fuzzer')
+  parser.add_argument('fuzzer_args', help='arguments to pass to the fuzzer',
+                      nargs=argparse.REMAINDER)
+  args = parser.parse_args(run_args)
+
+  if not _check_library_exists(args.library_name):
+    return 1
+
+  if not os.path.exists(os.path.join(BUILD_DIR, 'out', args.library_name,
+                                     args.fuzzer_name)):
+    print(args.fuzzer_name,
+          'does not seem to exist. Please run build_fuzzers first.',
+          file=sys.stderr)
+    return 1
+
+  temp_dir = tempfile.mkdtemp()
+
+  command = [
+      'docker', 'run', '-i',
+      '-v', '%s:/out' % os.path.join(BUILD_DIR, 'out'),
+      '-v', '%s:/cov' % temp_dir,
+      '-w', '/cov',
+      '-e', 'ASAN_OPTIONS=coverage=1,detect_leaks=0',
+      '-t', 'ossfuzz/libfuzzer-runner',
+      '/out/%s/%s' % (args.library_name, args.fuzzer_name),
```

Comment:
I'm surprised by this additional directory. AFAIK our build scripts do not create second level.